css/legacy.css

```diff
@@ -0,0 +1,2 @@
+#bootstrap-theme .crm-mosaico-page .helpicon{background-color:#42afcb}
```

Comment:
This is not same as here https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/4/files#diff-1e6bdecc7a4b9016c394ff3e70c9209dL2
You added space between this two selectors `#bootstrap-theme .crm-mosaico-page`, check if we really need this